src/screens/fareResults.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import { ScreenNavigationProps } from '../routes';
+import { config } from '../config';
+import FlatListComponent from '../components/flatList';
+import { useEffect } from 'react';
+import { FareResponse, Journey } from '../model/fareResponseClass';
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: '#eee',
+  },
+  text: {
+    paddingBottom: 24,
+  },
+});
+
+type FareResultsProps = ScreenNavigationProps<'FareResults'>;
+
+const getTrainFares = async (
+  departStation: string | null,
+  arriveStation: string | null,
+): Promise<FareResponse> => {
+  const response: Response = await fetch(
+    `${config.baseURL}/v1/fares?originStation=${departStation}&destinationStation=${arriveStation}&outboundDateTime=2022-07-14T12:16:27.371&numberOfChildren=0&numberOfAdults=1`,
```

Comment:
Would be nice to pull this string construction out into a separate method.

No need to do this now, but you see on the main project we have a nice generic method to construct a string from a set of objects. For now, it's fine to just move this string formatting into its on method so for a bit of separation of concerns